### PR TITLE
Add information about supported key types for SSL certificates

### PR DESCRIPTION
**Pull request by mcalmer**
# Description

The SSL Cert deploy tool was enhanced to support EC keys. This PR add information about supported key types in SSL certificates.

# Target branches

Which documentation version does this PR apply to?

- [x] Master (Default)
- [x] Manager-4.3
- [ ] Manager-4.2

# Links

Master PR https://github.com/uyuni-project/uyuni/pull/7266
4.3 PR - https://github.com/SUSE/spacewalk/pull/22012

### CODE REVIEW

File: modules/administration/pages/ssl-certs-imported.adoc [hunk 1]
```diff
@@ -18,6 +18,7 @@ All files must be in PEM format.
 The host name of the SSL server certificate must match the fully qualified host name of the machine you deploy them on.
 You can set the host names in the ``X509v3 Subject Alternative Name`` section of the certificate.
 You can also list multiple host names if your environment requires it.
+Supported Key types are `RSA` and `EC` (Elliptic Curve).
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs-imported.adoc:21]:
```suggestion
Supported Key types are [literal]``RSA`` and [literal]``EC`` (Elliptic Curve).
```

File: modules/administration/pages/ssl-certs-imported.adoc [hunk 2]
```diff
@@ -18,6 +18,7 @@ All files must be in PEM format.
 The host name of the SSL server certificate must match the fully qualified host name of the machine you deploy them on.
 You can set the host names in the ``X509v3 Subject Alternative Name`` section of the certificate.
```
Comment (keichwa) [inline modules/administration/pages/ssl-certs-imported.adoc:19]:
```suggestion
You can set the host names in the [literal]``X509v3 Subject Alternative Name`` section of the certificate.
```

File: .changelog [hunk 3]
```diff
@@ -4,6 +4,7 @@
 # - Fixed error in Bat section of Upgrade Guide (bsc#1234567)
 # For guidelines: https://en.opensuse.org/openSUSE:Creating_a_changes_file_(RPM)
 
+- Add information about supported key types for SSL certificates
```
Comment (keichwa) [inline .changelog:7]:
```suggestion
- Listed supported key types for SSL certificates in Import SSL Certificates section of the
  Administation Guide.
```

**Review by keichwa — APPROVED** [review]
Cosmetic and drive-by suggestions.

**Comment by mcalmer** [comment]
Both Code PRs are merged. Docs can be included into 4.3 and master

**Review by jcayouette — APPROVED** [review]

**Pull request merged**